cfme/tests/services/test_catalog_item.py

```diff
@@ -134,10 +134,8 @@ def test_add_button(catalog_item, appliance):
     """
     button_name = catalog_item.add_button()
     view = appliance.browser.create_view(BaseLoggedInPage)
-    if appliance.version.is_in_series('5.8'):
-        message = 'Button "{}" was added'.format(button_name)
-    else:
-        message = 'Custom Button "{}" was added'.format(button_name)
+
+    message = 'Custom Button "{}" was added'.format(button_name)
```

Comment:
**Suggestion:** 
@dgaikwad Instead of `.format`, can you please use `f'` string .?
Like this --> `message = f'Custom Button "{button_name}" was added'`

Reply:
Done.